docs/docs/sourcing-from-buttercms.md

```diff
@@ -289,7 +283,7 @@ Then click on Create Page Type and name it "Customer Case Study". This will allo
 
 ![saving](https://buttercms.com/static/images/docs/guides/PagesNewPageType3.png)
 
-## Populate the content
+#### Populate the content
 
 Then populate our new page with content. In the next step, we'll call the ButterCMS API to retrieve this content from our app.
```

Comment:
```suggestion
Then populate the new page with content. In the next step, you'll call the ButterCMS API to retrieve this content from your app.
```
I just saw this when looking at the surrounding content structure (there may be other Style Guide fixes for this in a separate PR)